tools/cmake/mbed_greentea.cmake

```
@@ -70,6 +79,35 @@ macro(mbed_greentea_add_test)
 
     mbed_set_post_build(${MBED_GREENTEA_TEST_NAME})
 
+    if(NOT ${MBED_OUTPUT_EXT} STREQUAL "")
+        set(MBED_GREENTEA_TEST_IMAGE_NAME "${MBED_GREENTEA_TEST_NAME}.${MBED_OUTPUT_EXT}")
+    else()
+        set(MBED_GREENTEA_TEST_IMAGE_NAME "${MBED_GREENTEA_TEST_NAME}.bin")
+    endif()
+
+    if(DEFINED MBED_GREENTEA_HOST_TESTS_DIR)
+        list(APPEND MBED_HTRUN_ARGUMENTS "-e;${MBED_GREENTEA_HOST_TESTS_DIR}")
+    endif()
+
+    if(DEFINED MBED_TARGET)
+        list(APPEND MBED_HTRUN_ARGUMENTS "-m;${MBED_TARGET}")
+    endif()
+
```

Now we need to get https://github.com/ARMmbed/mbed-tools/pull/301 in and add
```suggestion

    if(NOT "${MBED_GREENTEA_TEST_RESET_TIMEOUT}" STREQUAL "")
        list(APPEND MBED_HTRUN_ARGUMENTS "-R;${MBED_GREENTEA_TEST_RESET_TIMEOUT}")
    endif()
```

---

Added.